portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -36,19 +37,30 @@
 
   private List<String> messages;
 
+  public static LinkedHashMap<String, Integer> commentsByPerson = new LinkedHashMap<>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query("Comment"); //.addSort("timestamp", SortDirection.DESCENDING);
+    Query query = new Query("Comment");
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
-    List<String> comments = new ArrayList<>();
+    List<String> comments = new ArrayList<String>();
     for (Entity entity : results.asIterable()) {
         long id = entity.getKey().getId();
         String nickname = (String) entity.getProperty("nickname");
         String email = (String) entity.getProperty("email");
         String text = (String) entity.getProperty("text");
         long timestamp = (long) entity.getProperty("timestamp");
         comments.add("Comment: " + text + "  |  Username: " + nickname + "  |  Email: "+email);
+        
+        if(commentsByPerson.containsKey(nickname)){
+             int temp = commentsByPerson.get(nickname);
+             commentsByPerson.put(nickname,temp+1);
+        }
```

Comment:
Nit: the format style recommended is 

if(...) {
  ...
} else {
  ...
}

Reply:
I have change this format for the if/else.    Does the Format also apply to for loops? e.g :
for(...){
...
} .. next line of code

And what does Nit stand for?

Reply:
It doesn't apply to for loops because the next line of code is not part of the for loop.

For if-else, we keep the lines close as they are one logic statement, e.g. else itself doesn't work. 

Nit is used for minor comments, or small bugs.

Reply:
Okay thank you for the clarification! This is done now.